style.css

```diff
@@ -363,45 +272,381 @@ li {
+    gap: 20px;
+    flex-wrap: wrap;
+    margin: 0 20px;
+    margin-bottom: 40px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    margin-top: 10%;
+  }
+
+  .description-info {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .project img {
+    display: flex;
+    width: 100%;
+  }
+
+  .content1 {
+    display: flex;
+    background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.103) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("ressources/Ilustration/Prj1.png");
+    background-size: cover;
+    background-position: center;
+    position: relative;
+    width: 369px;
+    height: 435px;
+    justify-content: flex-start;
+    align-items: flex-end;
+    padding: 1.5%;
+  }
+
+  .content2 {
+    display: flex;
+    background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.103) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("ressources/Ilustration/Pr2.png");
+    background-size: cover;
+    background-position: center;
+    position: relative;
+    width: 369px;
+    height: 435px;
+    justify-content: flex-start;
+    align-items: flex-end;
+    padding: 1.5%;
+  }
+
+  .content3 {
+    display: flex;
+    background: linear-gradient(180.45deg, rgba(38, 38, 38, 0.103) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("ressources/Ilustration/Pr3.png");
+    background-size: cover;
+    background-position: center;
+    position: relative;
+    width: 369px;
+    height: 435px;
+    justify-content: flex-start;
+    align-items: flex-end;
+    padding: 1.5%;
+  }
+
+  .about-me {
+    background-color: #1c1a19;
+    flex-direction: column;
+    background-image: url("ressources/Ilustration/illustration-about-desktop.png");
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    background-size: 50%;
+    padding: 5%;
+  }
+
+  .line {
+    width: 90%;
+  }
+
+  .skills--title-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    background-color: #1c1a19;
+    padding: 100px 6.4%;
+  }
+
+  .skills-title {
+    font-size: 32px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 70px;
+    color: #fff;
+    font-weight: 700;
+    font-family: "Inter", sans-serif;
+    margin-bottom: 10%;
+  }
+
+  .footer {
+    display: flex;
+  }
+
+  .titleFooter {
+    font-family: "Inter", serif;
+    font-weight: 700;
+    font-size: 32px;
+    width: 29%;
+    margin: 10%;
+    line-height: 44px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .form {
+    min-width: 300px;
+    margin: 10% 0%;
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+    gap: 20px;
+  }
+
+  .botn {
+    background: #ff6b00;
+    border-color: #ff6b00;
+    font-family: "inter", serif;
+    font-size: 17px;
+    font-weight: bold;
+    color: white;
+    height: 40px;
+    width: 130px;
+  }
+}
```

Comment:
please I'd suggest you increase the widht of the input boxes in your contact section so it''d be able to accept users details and it will look better than what we currently have 😄 

Your design :
![image](https://github.com/AndresEspi/portafolio/assets/20537712/d580465d-f94b-40b3-becf-e1f915876aac)

Figma design :
![image](https://github.com/AndresEspi/portafolio/assets/20537712/5e9bc043-f752-4eac-997d-0976944b9845)